.changeset/nasty-carpets-design.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/polaris': patch
```

Comment:
I'm not sure but changeset considered this a change in /polaris as well as tokens? Is this necessary or should I remove?

Reply:
That is odd. I would think Polaris would use the updated `@shopify/polaris-tokens` bump since we use the `*` version range.